tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -1023,9 +1023,7 @@ type ProjectWorkflowBuilder
 
     member this.DeleteProjectDir() =
         if Directory.Exists initialProject.ProjectDir then
-            try
-                Directory.Delete(initialProject.ProjectDir, true)
-            with _ -> ()
+            () //try Directory.Delete(initialProject.ProjectDir, true) with _ -> ()
```

Comment:
?

Reply:
That is temporary, although I think I will add an an environment variable to turn off deletes.